### PR TITLE
Store index

### PR DESCRIPTION
**Pull request by vaidashi**
LIST OF CHANGES:
-As a guest customer, I can add items from multiple businesses into a single cart. 
- Routes now nest items under stores
-
-

THINGS I'M UNSURE ON:
- factory girl was being a pain, may need to change some of those in the near future
-
-

RSPEC GREEN?
**YES**
NO- WHY?

*MENTION GROUP MEMBER TO REVIEW PR*
@jimszalew @mimilettd

### CODE REVIEW

File: spec/factories/stores.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
 FactoryGirl.define do
   factory :store do
-    name "MyString"
-    description "MyString"
-    image "MyString"
+    sequence(:name) do |n|
+      "Store#{n}"
+    end
```
Comment (jimszalew) [inline spec/factories/stores.rb:5]: good stuff here, more flexibility with factory girl means easier test writing. Ash tested, Sad Path approved

**Review by jimszalew — APPROVED** [review]
This all looks pretty solid. Good skinny controllers

File: app/views/stores/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<ul>
+
+<% @stores.each do |store| %>
+<div class="store">
+  <li class="store_image-#{store.id}"> <%= image_tag store.image %> </li>
+  <li class="store-#{store.id}"> <%= link_to "#{store.name}", store_path(store) %> </li>
+</div>
+   <% end %>
+ </ul>
```
Comment (mimizotti) [inline app/views/stores/index.html.erb:9]: Love the css classes on these. Good job!

File: app/views/stores/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <% if item.active? %>
+          <%= button_to "Add to Cart", cart_path(item_id: item.id), class: 'btn btn-secondary add-to-cart' %>
+        <% else %>
+          <p>Item Retired</p>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
+<% end %>
```
Comment (mimizotti) [inline app/views/stores/show.html.erb:22]: This is solid. Good work.

**Review by mimizotti — APPROVED** [review]
Thanks for completing this feature @vaidashi! Looks good.

**Pull request merged**